app/models/user.rb

```diff
@@ -111,6 +111,12 @@ def invitation_status
     "invited" if invitation_sent_at.present?
   end
 
+  def mark_invitation_status_as_accepted!
```

Comment:
❓Question: Another possible solution would be to implement this inside the seeds file instead of the `User` model class. I find it clearer this way, but the `User` class is at risk of becoming a "monster class," and this addition might encourage others to add even more complexity to it. Do you think it's okay as it is?

Reply:
No, I would move this method to the seeds file.

Reply:
No, I would move this method to the seeds file.

Reply:
Alright @dorner, I've just moved the method to the seeds file and added the mentioned comment.